.travis.yml

```diff
@@ -43,6 +43,7 @@ env:
     - KONG_TEST_DATABASE=postgres TEST_SUITE=plugins
     - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=plugins
     - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=plugins
+    - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=external_plugins
```

Comment:
Missing jobs for Postgres and C* 2.2?

Reply:
maybe? Some of the plugins don't exercise a datastore at all some of them are hardcoded to cassandra.

No harm in adding 2.2 I suppose

Reply:
All plugins exercise a datastore (outside of dbless, of course) at least for the plugin configuration. Granted, some tests are often written specifically for a given storage strategy, but that's not a universal truth - it's handled in the test suite itself.

Reply:
I think this shouldn't be a separate job but instead be included in the `plugins` job alongside the in-repo plugins. This way they will be tested will all DB combinations.

Reply:
I partly separated them for CI parallelization speed but fine with it being merged into `plugins` too